libpod/runtime.go

```diff
@@ -221,6 +221,9 @@ type RuntimeConfig struct {
 	// NumLocks is the number of locks to make available for containers and
 	// pods.
 	NumLocks uint32 `toml:"num_locks,omitempty"`
+
+	// EventsLogFilePath is where the events log is stored.
+	EventsLogFilePath string `toml:"events_logfile_path"`
```

Comment:
Make this `toml:-` - I don't really want people messing with this via config file